README.md

```diff
@@ -1,189 +1,249 @@
 
-```set C2PA_PRIVATE_KEY=$(cat my_es256_private_key)```
-```set C2PA_PUB_CERT=$(cat my_es256_certs)```
+```shell
+set C2PA_PRIVATE_KEY=$(cat my_es256_private_key)
+set C2PA_PUB_CERT=$(cat my_es256_certs)
+```
 
-The both private key and sign cert should be in PEM format.  The sign cert should contain a certificate
-chain PEMs starting for the end-entity certificate used to sign the claim ending with intermediate certificate
-before the root CA certificate.  See ```sample`` folder for example certificates.
+Both the `private_key` and `sign_cert` should be in PEM format. The `sign_cert` should contain a PEM certificate chain starting for the end-entity certificate used to sign the claim ending with the intermediate certificate before the root CA certificate.  See the "sample" folder for example certificates.
 
-To create your own temporary files for testing you can execute the following command
+To create your own temporary files for testing, you can execute the following command:
 
 ```shell
-sudo openssl req -new -newkey rsa:4096 -sigopt rsa_padding_mode:pss -days 180 -extensions v3_ca -addext "keyUsage = digitalSignature" -addext "extendedKeyUsage = emailProtection" -nodes -x509 -keyout private.key -out certs.pem -sha256
+sudo openssl req -new -newkey rsa:4096 
+   -sigopt rsa_padding_mode:pss \ 
+   -days 180 \
+   -extensions v3_ca \
+   -addext "keyUsage = digitalSignature" \
+   -addext "extendedKeyUsage = emailProtection" \
+   -nodes -x509 -keyout private.key -out certs.pem -sha256
 ```	
 
-Note you may have need to update your openssl version if the above command does not work.
+**Note:** You may have need to update your `openssl` version if the above command does not work. You will likely need version 3.0 or later. You can check the version that is installed by typing `openssl version`.
 
-c2patool can also timestamp the signature data that is embedded.  This is useful for validating an asset when the embedded 
-certificates have expired.  If the config has a ta_url set, c2patool will attempt to timestamp the signature using the TA service at the provided URL.  The TA must be RFC3161 compliant.  Example TA setting:
+c2patool can also timestamp the embedded signature data.  This is useful for validating an asset when the embedded certificates have expired. If the config has a `ta_url` set, c2patool will attempt to timestamp the signature using the TA service at the provided URL. The TA must be [RFC3161](https://datatracker.ietf.org/doc/html/rfc3161) compliant. For example,
 
-```ta_url=http://timestamp.digicert.com```
+```shell
+ta_url=http://timestamp.digicert.com
+```
 
-## Configuration file format
+##### Manifest definition format
 
-The Configuration file is a JSON formatted file with a .json extension:
+The manifest definition file is a JSON serialized file (`.json`). The schema for this type is as follows:
```

Comment:
We should move this config definition below the rest of the commands and reference it.

Reply:
@gpeacock Should we make this its own file that we can point to so we don't have to have a big blob of JSON inline?

Reply:
yes, but we will need to be clear about referencing it, since it documents a lot that is not documented elsewhere. We should likely have more examples in the docs.